release/context.go

```diff
@@ -187,3 +193,106 @@ func (rc *ReleaseContext) FindDefinedServices() ([]*ServiceUpdate, error) {
 	}
 	return defined, nil
 }
+
+type FilterResult struct {
+	Result flux.ServiceResult
```

Comment:
Due to top level package naming. ;-)
Renamed to ServiceResult, but not much more I can do.